tests/test_layout_objects.py

```diff
@@ -260,8 +262,15 @@ def test_inline_checkboxes_failing(self):
             expected = "inline_checkboxes_failing.html"
         assert parse_form(form) == parse_expected(expected)
 
+    @pytest.mark.parametrize("keep_compatibility", [
+        True,
+        pytest.param(False, marks=pytest.mark.xfail(
+            condition=crispy_forms.__version__ <= "2.2",
+            reason="django-crispy-forms issue #1395",
+        )),
+    ])
```

Comment:
Then i was making these tests a little bit more complicated , i was thought we run these tests in couple with several versions of django-crispy-forms :) But actually, it is not.
We should to choose one thing: Test template with different versions of base lib, or make revert updated tests to initial state.
Probably to test templates with different versions of the base lib is not a bad idea, what do you think?

Reply:
Nice. I was thinking about this. 

I was thinking we should bump the minimum supported version of django-crispy-forms. What do you think?

Reply:
Good proposal. It's done ✅